src/cares_wrap.cc

```diff
@@ -1266,8 +1266,14 @@ static void Initialize(Handle<Object> target,
               Integer::New(env->isolate(), AF_UNSPEC));
   target->Set(FIXED_ONE_BYTE_STRING(env->isolate(), "AI_ADDRCONFIG"),
               Integer::New(env->isolate(), AI_ADDRCONFIG));
+#ifdef __FreeBSD__
```

Comment:
Nope, it won't help. The source of the problem is here: https://github.com/iojs/io.js/blob/v1.x/lib/net.js#L894-L900